uvod-do-programovani-2/json/excs/zavod.md

```diff
@@ -3,11 +3,15 @@ title: Závod
 demand: 3
 ---
 
-Pracuj dál se souborem `zavod.json` a zjisti čas závodníka, který získal stříbrnou medaili. Dále projdi data pomocí cyklu a vytvoř seznam všech závodníků, kteří závod dokončili, tj. jejich oficiální čas není DNF.
+Pracuj dál se souborem [zavod.json](../assets/zavod.json). Cílem cvičení je zjistit čas závodníka, který získal stříbrnou medaili - seznam závodníků je seřazený, tedy výherce je zapsán jako první v našem souboru `zavod.json`. Budeš tedy muset projít data pomocí cyklu a vytvořit seznam všech závodníků, kteří závod dokončili, tj. jejich oficiální čas není `'DNF'`.
```

Comment:
Tady není jistý, že ten link bude fungovat. Po mergi bychom to museli vyzkoušet a udělat kdyžtak ještě fix.

Reply:
Souhlasím.

Reply:
Fixnuto v #99